### PR TITLE
Table captions

### PR DESCRIPTION
**Pull request by ethax-ross**
### Trello card

[Trello-3251](https://trello.com/c/mjikZkhz/3251-dac-audit-no-table-captions)

### Context

We want to ensure all tables have an associated caption for accessibility.

### Changes proposed in this pull request

- Expand markdown to enable table captions

Markdown/Kramdown has basic support for HTML tables, however it does not currently support captions.

Expand the `markdown_template_handler` to recognise and correctly insert captions into tables.

The handler looks for text proceeding the table in the format:

```
Table caption: My table caption
```

- Add captions to markdown tables

Add captions to all of the markdown tables. The HTML tables I found already had captions.

### Guidance to review

Pages with tables on them:

[Funding your training](https://review-get-into-teaching-app-2541.london.cloudapps.digital/funding-your-training)
[Salaries and benefits](https://review-get-into-teaching-app-2541.london.cloudapps.digital/salaries-and-benefits)
[Cookies](https://review-get-into-teaching-app-2541.london.cloudapps.digital/cookies)
[How to fund your teacher training](https://review-get-into-teaching-app-2541.london.cloudapps.digital/blog/how-to-fund-your-teacher-training)

### CODE REVIEW

File: app/views/content/funding-your-training.md [hunk 1]
```diff
@@ -131,6 +131,8 @@ below. You cannot receive both a teaching bursary and a scholarship.
 | **Maths**                                        | £24,000  | £26,000          |
 | **Physics**                                      | £24,000  | £26,000          |
 
+Table caption: Postgraduate teaching bursaries and scolarships by subject
```
Comment (gemmadallmandfe) [inline app/views/content/funding-your-training.md:134]:
```suggestion
Table caption: Postgraduate teaching bursaries and scholarships by subject
```

**Comment by gemmadallmandfe** [comment]
@ethax-ross Let's discuss this PR next week as it's been hanging around too long!

Looking at it again, this is not just about reviewing content around the caption - I raised some other questions above re:

a) visually hiding the captions?
b) if not, formatting them differently
c) the big string of numbers that VoiceOver reads out

**Comment by ethax-ross** [comment]
Closing as we are opting to not add table captions for the moment. We may revisit this later with adjusted copy/caption design.

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me from a front-end perspective - I have tested with VoiceOver

**Review by martyn-w — APPROVED** [review]
👍  LGTM

**Review by asatwal — APPROVED** [review]
👍 LGTM

**Pull request merged**